_layouts/default.liquid

```diff
@@ -0,0 +1,138 @@
+{%- assign lang = page.name | replace: '.md', '' -%}
+<!doctype html>
+<html lang="{{ lang }}">
+  <head>
+    {%- if page.name == 'index.md' -%}
+      <link rel="canonical" href="/en" />
+      <meta content="0; url=/en" http-equiv="refresh" />
+    {% endif %}
+    <meta charset="UTF-8" />
+    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible" />
+    <meta
+      content="initial-scale=1.0, maximum-scale=1.0, user-scalable=1, width=device-width"
+      name="viewport"
+    />
+    <meta content="{{ page.header.sub-title }}" name="description" />
+    <link
+      href="/assets/css/style.css" media="screen"
+      rel="stylesheet" type="text/css"
+    />
+    <title>{{ page.header.title }}</title>
+  </head>
+  <body class="layout">
+    <div class="layout--container">
+      <header class="layout--header">
+        <nav class="locales">
+          {%- assign pages = site.pages | sort: 'name' -%}
+          {%- for a_page in pages -%}
+            {%- assign a_page_ext = a_page.name | split: '.' | last -%}
+            {%- if a_page.name == 'index.md' or a_page_ext != 'md' -%}
+              {%- continue -%}
+            {%- endif -%}
+            {%- assign a_page_lang = a_page.name | replace: '.md', '' -%}
+            {%- if a_page_lang == lang -%}
+              <a class="locales--link -selected" href="/{{ a_page_lang }}">
+                {{- a_page_lang | upcase -}}
```

Comment:
I find duplicated what should be the same code (in `else`) odd here - couldn't we instead set an `additional_class` or something and interpolate that into the same `a` element to avoid duplication of the DOM structure and only changing what's really changing instead?

Reply:
Fair. Fixed!